sycl/test/kernel_from_file/hw.cpp

```diff
@@ -1,10 +1,10 @@
 // UNSUPPORTED: cuda
-// CUDA does not support SPIR-V.
+// CUDA does not support SPIRV.
```

Comment:
```suggestion
// CUDA does not support SPIR-V.
```
I think `SPIR-V` is the correct spelling of the device code format.

Reply:
You are right: https://www.khronos.org/registry/spir-v/
Got confused by this: https://github.com/KhronosGroup/SPIRV-Tools

Will change.